source/Mission.cpp

```diff
@@ -1300,6 +1302,12 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 	subs["<commodity>"] = result.cargo;
 	subs["<tons>"] = to_string(result.cargoSize) + (result.cargoSize == 1 ? " ton" : " tons");
 	subs["<cargo>"] = subs["<tons>"] + " of " + subs["<commodity>"];
+	subs["<outfit>"] = result.OutfitName();
+	subs["<outfit-units>"] = to_string(result.outfitUnits);
+	subs["<outfit-tons>"] = to_string(result.OutfitUnitsMassInt())
+		+ (result.OutfitUnitsMassInt() == 1 ? " ton" : " tons");
+	subs["<outfit-cargo>"] = (subs["<outfit-units>"] + " units of " + subs["<outfit>"]
```

Comment:
If 'subs["<outfit-units>"]' is 1, it would say '1 units of' instead of '1 unit of'.